ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/networking/libp2p/rpc/BlobSidecarsByRootRequestMessage.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright ConsenSys Software Inc., 2022
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.spec.datastructures.networking.libp2p.rpc;
+
+import static tech.pegasys.teku.spec.config.Constants.MAX_REQUEST_BLOB_SIDECARS;
+
+import java.util.List;
+import tech.pegasys.teku.infrastructure.ssz.SszList;
+import tech.pegasys.teku.infrastructure.ssz.impl.SszListImpl;
+import tech.pegasys.teku.infrastructure.ssz.schema.impl.AbstractSszListSchema;
+import tech.pegasys.teku.infrastructure.ssz.tree.TreeNode;
+
+public class BlobSidecarsByRootRequestMessage extends SszListImpl<BlobIdentifier>
+    implements SszList<BlobIdentifier>, RpcRequest {
+
+  public static class BlobSidecarsByRootRequestMessageSchema
+      extends AbstractSszListSchema<BlobIdentifier, BlobSidecarsByRootRequestMessage> {
+
+    public BlobSidecarsByRootRequestMessageSchema() {
+      // account for future forks by handling up tp to 128 MAX_BLOBS_PER_BLOCK
+      super(BlobIdentifier.SSZ_SCHEMA, MAX_REQUEST_BLOB_SIDECARS.times(128).longValue());
```

Comment:
I think we'd better move 128 to some constant for cleanliness and improve comment text with something like "actual size validation MUST BE done when used" or "actual size validation is done in message handler and .."

Reply:
Agree about the constant. Will improve that and the comment.